extensions/resteasy/runtime/src/main/java/io/quarkus/resteasy/runtime/standalone/VertxBlockingOutput.java

```diff
@@ -69,10 +69,17 @@ public void write(ByteBuf data, boolean last) throws IOException {
             //do all this in the same lock
             synchronized (request.connection()) {
                 awaitWriteable();
```

Comment:
This needs to be in the try block as it can now throw

Reply:
thanks, correcting